src/app/destinyTrackerApi/perkRater.js

```diff
@@ -0,0 +1,109 @@
+import _ from 'underscore';
+
+/**
+ * Rate perks on an item (based off of its attached user reviews).
+ *
+ * @class PerkRater
+ */
+class PerkRater {
+  /**
+   * Rate the perks on an item based off of its attached user reviews.
+   *
+   * @param {any} item
+   * @memberof PerkRater
+   */
+  ratePerks(item) {
+    if ((!item.writtenReviews) ||
+        (!item.writtenReviews.length)) {
+      return;
+    }
+
+    const maxColumn = this._getMaxColumn(item);
+
+    for (let i = 1; i < maxColumn; i++) {
+      const perkNodesInColumn = this._getPerkNodesInColumn(item, i);
+
+      const ratingsAndReviews = _.map(perkNodesInColumn, (perkNode) => this._getPerkRatingsAndReviewCount(perkNode, item.writtenReviews));
+
+      const maxReview = this._getMaxReview(ratingsAndReviews);
+
+      this._markNodeAsBest(maxReview);
+    }
+  }
+
+  _markNodeAsBest(maxReview) {
+    if (!maxReview) {
+      return;
+    }
+
+    maxReview.perkNode.bestRated = true;
+  }
+
+  _getMaxReview(ratingsAndReviews) {
```

Comment:
Most of these don't need to be member functions of a class (they don't use `this`).